### ISSUE TITLE
Accessibility: Press `/` to focus does not work with NVDA/Firefox

### ISSUE THREAD

**Issue by ghost** [opentofu/registry-ui#150]
The search box has a prompt to use `/` to focus to the search box, which does not seem to work in conjunction with the NVDA screen reader.

**Comment by diofeher** [comment]
Out of curiosity, how did you test the NVDA screen reader? Using something like Browserstack?

**Comment by diofeher** [comment]
@abstractionfactory I just tested here with Microsoft Edge and Firefox, and I could see the same bug. Typing / after page load doesn't focus on the search bar, but if you click on the search box, click outside, then type /, it works as expected.

**Comment by diofeher** [comment]
@abstractionfactory Hey, I just understood this issue, and I think it's just a matter of how NVDA works.

According to this answer: https://stackoverflow.com/a/52264128/289657, NVDA is on browser mode so keys are intercepted by the screen reader and don't work as expected.
There are two types of modes: browser (when you're navigating) and focus mode (when you're typing to interact with the application) .

<img width="1145" alt="Screenshot 2024-12-31 at 17 52 56" src="https://github.com/user-attachments/assets/77f038e0-649c-4e13-8dc6-40a344fc22af" />

If you switch to focus mode, then type slash, it works as expected, by focusing on the input. 
If it's on browser mode (the default when you open the page), the keys are intercepted and the slash is not sent to the application.

 So I would say there's no fix for this one because of the way NVDA works.

**Comment by diofeher** [comment]
From what I understood, I think there's no hotkey that could achieve that 

https://stackoverflow.com/questions/72538936/web-development-screen-reader-blocks-keyboard-events

"Several screen readers offer [two different modes of behavior](https://tink.uk/understanding-screen-reader-interaction-modes/), depending on whether the content is operable or not.

When some screen readerd encounter non-operable content, it enters "browse mode" (aka. "document mode" or "scan mode") 'takes control' of the keyboard, offering shortcuts for "next heading", "next word", "jump to navigation" and so on. In this mode, your own keyboard and mouse event handlers will never be triggered."
